target/linux/ramips/dts/R6120.dts

```diff
@@ -0,0 +1,146 @@
+/dts-v1/;
+
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "netgear,r6120", "mediatek,mt7628an-soc";
+	model = "Netgear AC1200 R6120";
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x4000000>;
+	};
+
+	gpio-keys-polled {
+		compatible = "gpio-keys-polled";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		poll-interval = <20>;
+
+		reset {
+			label = "reset";
+			gpios = <&gpio1 6 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+	};
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		lan {
+			label = "r6120:green:lan";
+			gpios = <&gpio1 12 GPIO_ACTIVE_LOW>;
+		};
+
+		power {
+			label = "r6120:green:power";
+			gpios = <&gpio1 11 GPIO_ACTIVE_LOW>;
+		};
+
+		wlan {
+			label = "r6120:green:wlan2g";
+			gpios = <&gpio1 10 GPIO_ACTIVE_LOW>;
+		};
+
+                wlan_orange {
+                        label = "r6120:orange:wlan2g";
+                        gpios = <&gpio1 9 GPIO_ACTIVE_LOW>;
+                };
+
+		wlan5 {
+			label = "r6120:green:wlan5g";
+			gpios = <&gpio1 8 GPIO_ACTIVE_LOW>;
+		};
+
+                wlan5_orange {
+                        label = "r6120:orange:wlan5g";
+                        gpios = <&gpio1 7 GPIO_ACTIVE_LOW>;
+                };
+
+	};
+};
+
+&pinctrl {
+	state_default: pinctrl0 {
+		gpio {
+			ralink,group = "i2c", "gpio", "p0led_an", "p1led_an", "p2led_an",
```

Comment:
drop the i2c and gpio pins from the group. none of the pins is used
![bildschirmfoto vom 2018-06-21 23-48-03](https://user-images.githubusercontent.com/4547948/41747392-2d6ed04e-75ae-11e8-8563-ad32c36c5e04.png)